src/components/bookappointment/modal/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { ThemeProvider } from '@emotion/react';
+import { Close } from '@mui/icons-material';
+import {
+  CssBaseline,
+  Container, Paper, IconButton, Typography, Stepper, Step, StepLabel, Box, Button, Modal,
+} from '@mui/material';
+import * as React from 'react';
+import { createTheme } from '@mui/material/styles';
+import { useFormik } from 'formik';
+import ModelStyle from '../../../pages/therapistProfile/classes';
+import BookAppointment from '../selectTime/index';
+import StripePaymentForm from '../payment';
+import ValidationSchema from './validation';
+
+const steps = ['select appointment', 'Payment details'];
+const defaultTheme = createTheme();
+
+const Comp = ({ step, formik }: any) => {

```

Comment:
```suggestion
const Comp = ({ step, formik }: any) => {

```
please change component name and replace any with the real type